packages/unmock-core/src/service/index.ts

```diff
@@ -1,16 +1,16 @@
-import { DEFAULT_ENDPOINT, DEFAULT_REST_METHOD } from "./constants";
+import { DEFAULT_ENDPOINT } from "./constants";
 import {
   HTTPMethod,
   IService,
   isRESTMethod,
-  IUnmockServiceState,
+  UnmockServiceState,
 } from "./interfaces";
 import { ServiceStore } from "./serviceStore";
 
 const saveStateProxy = (store: ServiceStore, serviceName: string) => (
   endpoint = DEFAULT_ENDPOINT,
-  method: HTTPMethod = DEFAULT_REST_METHOD,
-  state: IUnmockServiceState,
+  method: HTTPMethod = "all",
```

Comment:
Random comment: "all" isn't really an `HTTPMethod`, I guess? If that wasn't there, we could use this type in `ISerializedRequest.method` as well, I assume.

Reply:
It's part of `HTTPMethod`, but not an actual HTTP method :roll_eyes: 
It's used to signal that any method should match (perhaps `any` is better than `all`?)

Reply:
Yes I understand its purpose for the state management, just wondering what I should call the type for `method` in `ISerializedRequest` if `HTTPMethod` already exists.

Reply:
Hm, we could separate the state management then and just have a `type AugHTTPMethod = "all" | HTTPMethod`.